components/prow/manifests/nginx-controller.yaml

```diff
@@ -354,7 +354,7 @@ spec:
       paths:
       - backend:
           serviceName: prometheus-meta
-          servicePort: 80
+          servicePort: http
```

Comment:
This name should be `prom-web` because that name is used in the [prometheus-meta-service](https://github.com/prometheus/prombench/blob/23d6d4e5930ee7f92706ecea6c97854ae66c398a/components/prombench/manifests/results/prometheus-meta.yaml#L166) definition